ZukoAzula2016/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding//src/org/team1540/zukoazula/RobotTemplate.java=UTF-8
```

Comment:
Tyler's editor destroyed the emoji anyways, so I don't think that's necessarily true...